src/main/java/com/github/javafaker/service/FakeValuesService.java

```diff
@@ -169,6 +181,30 @@ public String letterify(String letterString) {
         return sb.toString();
     }
 
+    /**
+     * Returns a string with the '?' characters in the parameter replaced with random alphabetic
+     * characters.
+     * <p/>
+     * For example, the string "12??34" could be replaced with a string like "12AB34".
+     *
+     * @param letterString
+     * @param isUpper specifies whether or not letters should be upper case
+     * @return
+     */
+    public String letterify(String letterString, boolean isUpper) {
+        StringBuffer sb = new StringBuffer();
+	int baseChar = (isUpper) ? 65 : 97; // from ascii table
```

Comment:
could you please perform an extract method and put the common parts in one method that both letterify methods call? Thanks